server/src/main/java/io/spine/system/server/SystemEnricher.java

```diff
@@ -47,25 +47,26 @@ private SystemEnricher() {
     /**
      * Creates a new {@link Enricher} for the system bounded context.
      *
-     * @param commandRepository repository to find enrichment values in
+     * @param repo the repository for obtaining scheduled command instances
      * @return new {@link Enricher}
      */
-    public static Enricher create(CommandLifecycleRepository commandRepository) {
-        checkNotNull(commandRepository);
-        Enricher enricher = Enricher
+    public static EventEnricher create(CommandLifecycleRepository repo) {
+        checkNotNull(repo);
+        EventEnricher enricher = EventEnricher
                 .newBuilder()
-                .add(CommandId.class, Command.class, commandLookup(commandRepository))
+                .add(CommandScheduled.class, Command.class,
```

Comment:
Do we really need to place the last parameter on the new line? It looks like it should fit.